### ISSUE TITLE
Best way to implement MixtureLogistic loss with tfp in VAE setting

### ISSUE THREAD

**Issue by giovp** [tensorflow/probability#1051]
Hi,

I would be interested in using a discretized mixture of logistic layer for a Decoder in a VAE settings (similar to [NVAE](https://arxiv.org/pdf/2007.03898.pdf)). Exploring the tfp API, it seems like the most appropriate layer would be the [MixtureLogistic layer](https://www.tensorflow.org/probability/api_docs/python/tfp/layers/MixtureLogistic). Does it make sense? Would it work in tf.keras model built with keras functional API?
I will just try but wanted to ask whether you would suggest any other approach, or whether, for that purpose, I should go on and re-write the layer in TF (saw some implementations in pytorch around).

Thank you for any feedback!
